File: app/src/main/java/com/example/webviewjava/MainActivity.java
package com.example.webviewjava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void openCamera(View view){
        Intent i = new Intent(this, CameraActivity.class);
        startActivity(i);
    }

    public void getLocation(View view){
        Intent i = new Intent(this, Location.class);
        startActivity(i);
    }

    public void uploadFile(View view){
        Intent i = new Intent(this, FileUpload.class);
        startActivity(i);
    }


}

File: app/src/main/java/com/example/webview/MainActivity.kt
package com.example.webview

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val fab: View = findViewById(R.id.cameraButton)
        fab.setOnClickListener { view ->
            val i = Intent(this, WebViewActivity::class.java)
            startActivity(i)
        }
        val locationButton: View = findViewById(R.id.locationButton)
        locationButton.setOnClickListener { view ->
            val i = Intent(this, LocationActivity::class.java)
            startActivity(i)
        }
        val fileUpload: View = findViewById(R.id.fileUploadButton)
        fileUpload.setOnClickListener { view ->
            val i = Intent(this, FileUploadActivity::class.java)
            startActivity(i)
        }
    }

}